### DISCUSSION TITLE
was supabase necessary in this project?

### DISCUSSION THREAD

**Discussion by Greatz08** [AykutSarac/jsoncrack.com#401] [General]
Asking this because supabase is db+analytical thing and do we need that in this project.As a selfhosted I wanted things as light as possible so can't it run without it and can't postgres only be used?

**Comment by AykutSarac** [comment]
Supabase and Google Analytics can be disabled manually through the code, they are not necessary for visualization and light usage overall.

**Comment by AykutSarac** [comment]
Supabase has been removed from the project, as cloud related functions have been moved to ToDiagram.
